### PR TITLE
OKTA-756629 - Update CAE policies vuepress

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** To avoid confusion, updating the CAE polices on vuepress Policy API reference to use the Post auth session evaluation name.
- **Is this PR related to a Monolith release?** `2024.08.0`

### Resolves:

* [OKTA-756629](https://oktainc.atlassian.net/browse/OKTA-756629)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/policies/index.md](https://github.com/okta/okta-developer-docs/blob/9be835454f9ad063476d9c7aeb3b6e3a90b64bd8/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fpolicies%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9b74368e-5704-4bcf-9f39-df41b8de85bb) | 62  | 82  | 66  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.e9720b9f-9863-4897-8aab-1b339cb22a68?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by ericalong-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 1]
```diff
@@ -851,7 +851,7 @@ The Policy object defines several attributes:
...
-| type        | Specifies the [type of Policy](#policy-types). Valid values: `OKTA_SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, or `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `CONTINUOUS_ACCESS`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on the Identity Engine.  | String                                            | Yes      |                        |
+| type        | Specifies the [type of Policy](#policy-types). Valid values: `OKTA_SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, or `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `POST_AUTH_SESSION`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on the Identity Engine.  | String                                            | Yes      |                        |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:854]:
```suggestion
| type        | Specifies the [type of Policy](#policy-types). Valid values: `OKTA_SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, or `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `POST_AUTH_SESSION`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on Identity Engine.  | String                                            | Yes      |                        |
```

File: packages/@okta/vuepress-site/docs/reference/api/policy/index.md [hunk 2]
```diff
@@ -970,7 +970,7 @@ The Rules object defines several attributes:
...
 | id            | Identifier of the Rule                                             | String                                         | No         | Assigned               |
-| type          | Rule type. Valid values: `SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, and `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `CONTINUOUS_ACCESS`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on the Identity Engine. <br><br> | String (Enum)                                 | Yes        |                        |
+| type          | Rule type. Valid values: `SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, and `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `POST_AUTH_SESSION`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on the Identity Engine. <br><br> | String (Enum)                                 | Yes        |                        |
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/policy/index.md:973]:
```suggestion
| type          | Rule type. Valid values: `SIGN_ON`, `PASSWORD`, `MFA_ENROLL`, and `IDP_DISCOVERY`.<br><br> <ApiLifecycle access="ie" /><br>**Note:** The following policy types are available only with the Identity Engine: `ACCESS_POLICY`, `PROFILE_ENROLLMENT`, `POST_AUTH_SESSION`, and `ENTITY_RISK`. <br>[Contact support](https://support.okta.com/) for more information on Identity Engine. <br><br> | String (Enum)                                 | Yes        |                        |
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/concepts/policies/index.md](https://github.com/okta/okta-developer-docs/blob/dac69d310efaaf93e92128ddc0bbc5a5c627ea8a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fconcepts%2Fpolicies%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fdf1ad5f-878b-426f-9b0b-54589277253f) | 62  | 82  | 66  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.09364b69-d0d0-46c1-84fd-88ac2c7f3531?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**